python/ray/_private/worker.py

```diff
@@ -2777,22 +2778,61 @@ def remote(
 
 
 @PublicAPI
-def remote(*args, **kwargs):
+def remote(
+    *args, **kwargs
+) -> Union[ray.remote_function.RemoteFunction, ray.actor.ActorClass]:
     """Defines a remote function or an actor class.
 
-    This can be used with no arguments to define a remote function or actor as
-    follows:
+    This function can be used as a decorator with no arguments
+    to define a remote function or actor as follows:
 
     .. code-block:: python
```

Comment:
Can we make sure these docstrings run in the CI? See https://github.com/ray-project/ray/pull/28421

Reply:
Basically you need to replace `.. code-block:: python` with `>>> ` in front of every line -- examples in the linked PR